package.json

```diff
@@ -1,22 +1,44 @@
 {
-  "name": "pizza1",
-  "version": "1.0.0",
-  "description":"CYBERPIZZA",
-  "main": "index.js",
-  "scripts": {
-    "start": "node server"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/AStashevskaya/cyberpizza.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/AStashevskaya/cyberpizza/issues"
-  },
-  "homepage": "https://github.com/AStashevskaya/cyberpizza#readme",
-  "dependencies": {
-    "express": "^4.17.1"
-  }
+    "name": "pizza1",
+    "version": "1.0.0",
+    "description": "CYBERPIZZA",
+    "main": "index.js",
+    "scripts": {
+        "start": "webpack serve",
+        "lient:watch": "webpack --mode development --watch --open",
+        "client:build": "webpack --mode production",
+        "server:watch": "nodemon server/index.js --watch"
```

Comment:
@AStashevskaya Lets use `nodemon server --watch server` to avoid server reloading if client code is changed